src/error.rs

```diff
@@ -1155,8 +1155,24 @@ signed, unsigned, big, little, native, size, unit",
                     )
                     .unwrap();
                 }
-            },
 
+                RecordUpdateInvalidConstructor { location } => {
+                    let diagnostic = Diagnostic {
+                        title: "Invalid record constructor".to_string(),
+                        label: "This is not a record constructor".to_string(),
+                        file: path.to_str().unwrap().to_string(),
+                        src: src.to_string(),
+                        location: location.clone(),
+                    };
+                    write(buffer, diagnostic, Severity::Error);
+
+                    writeln!(
+                        buffer,
+                        "You are attempting to update a record using a value that is not a record constructor",
```

Comment:
I'm not a fan of this error message. Do you have any suggestions? This error is returned if something else is used with the record update syntax, like a normal function call, or a variable.